ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -108,15 +108,16 @@ pipeline {
             }
         }
 
-        stage('Boot VMs 1.13'){
+        stage('Boot VMs 1.{13,15}'){
```

Comment:
Can you put provisioning vms and running tests in parallel stages with separate directories (same way as for 1.11 and 1.12 above)?

Reply:
My bad, I thought my PR got merged before.